TestCase/Managers/InputManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Runtime.Intrinsics.X86;
+
+namespace TestCase.Managers
+{
+    public class InputManager
+    {
+        public int GetInput()
+        {
+            Console.Write("\nEnter number between 1 and 3: ");
+
+            if (int.TryParse(Console.ReadLine(), out int num))
+            {
+                if (num >= 1 && num <= 3)
+                {
+                    return num;
+                }
+                else
+                {
+                    Console.WriteLine("\nyou must give a number between 1 and 3");
+                    return GetInput();
+                }
+            }
+            Console.WriteLine("\nPlease enter a number between 1 and 3");
+            return GetInput();
+        }
+
+        public bool ResetChoiceInput()
```

Comment:
Method name could be better here. Do we need to repeat 'input' when this mangers responsibility is input? Maybe something like `ResetGame()`? Thoughts?